polaris-react/src/components/Button/Button.scss

```diff
@@ -30,14 +30,7 @@
 
   #{$se23} & {
     // stylelint-disable -- Polaris component custom properties
-    --pc-button-shadow: inset 0px -1px 0px #b5b5b5, inset -1px 0px 0px #e3e3e3,
-      inset 1px 0px 0px #e3e3e3, inset 0px 1px 0px #e3e3e3;
-    --pc-button-shadow-hover: inset 0px -1px 0px #cccccc,
```

Comment:
Perhaps we should leave this for post se23. I'm seeing ~8 usages and overrides outside of Polaris.